api/desecapi/tests/base.py

```diff
@@ -751,7 +751,6 @@ def requests_desec_domain_creation(cls, name=None):
             cls.request_pdns_zone_create(ns='LORD'),
             cls.request_pdns_zone_create(ns='MASTER'),
             cls.request_pdns_zone_axfr(name=name),
-            #cls.request_pdns_zone_retrieve(name=name),  # TODO not required anymore because ...?
```

Comment:
hm. I would have bet that we did not have this before either.

Reply:
we did that before to get the NS records, I believe